Resources/views/Button/create_button.html.twig

```diff
@@ -11,20 +11,27 @@ file that was distributed with this source code.
 
 {% if admin.hasAccess('create') and admin.hasRoute('create') %}
     {% if admin.subClasses is empty %}
-        <li>
-            <a class="sonata-action-element" href="{{ admin.generateUrl('create') }}">
+        <a class="sonata-action-element" href="{{ admin.generateUrl('create') }}">
+            <i class="fa fa-plus-circle" aria-hidden="true"></i>
+            {{ 'link_action_create'|trans({}, 'SonataAdminBundle') }}</a>
+    {% else %}
+        {% if 1 is same as (admin.subclasses|length) %}
```

Comment:
Please don't use Yoda style here, you don't risk assigning anything by mistake here.